src/main/java/seedu/address/model/Model.java

```diff
@@ -123,6 +123,19 @@ public interface Model {
      */
     void setVolunteer(Volunteer target, Volunteer editedVolunteer);
 
+    /**
+     * Adds the given pair.
+     * {@code pair} must not already exist in the address book.
```

Comment:
```suggestion
     * {@code pair} must not already exist in FriendlyLink.
```

Reply:
Resolved in commit 71e90f4f6ec2f74d637d12a4c6f48d26cb9ecb87